specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -787,41 +912,108 @@
             "name": "OperationalStatus",
             "modelAsString": true
           },
-          "description": "Operational status of the Disk pool"
+          "description": "Operational status of the Disk Pool."
         },
         "disks": {
           "type": "array",
-          "description": "List of Azure managed disks to attach to a DiskPool",
+          "description": "List of Azure Managed Disks to attach to a Disk Pool.",
           "items": {
             "$ref": "#/definitions/Disk"
           },
-          "minLength": 1,
-          "maxLength": 15
+          "minLength": 0,
+          "maxLength": 8
         },
         "subnetId": {
           "type": "string",
-          "description": "Azure resource id of the subnet for the DiskPool",
+          "description": "Azure Resource ID of a Subnet for the Disk Pool.",
           "x-ms-mutability": [
             "create",
             "read"
           ]
+        },
+        "additionalCapabilities": {
+          "type": "array",
+          "description": "List of additional capabilities for Disk Pool.",
+          "items": {
+            "$ref": "#/definitions/AdditionalCapability"
+          }
+        }
+      }
+    },
+    "DiskPoolCreateProperties": {
+      "type": "object",
+      "description": "Properties for Disk Pool create or update request.",
+      "required": [
+        "availabilityZones",
+        "subnetId"
+      ],
+      "properties": {
+        "availabilityZones": {
+          "type": "array",
+          "description": "Logical zone for Disk Pool resource; example: [\"1\"].",
```

Comment:
Could user specify multiple zones for one diskpool?

Reply:
No, but we valdidate that there's only one zone passed in on the server side. We modeled this after what [Azure Kubernetes Service ](https://github.com/Azure/azure-rest-api-specs/blob/44ec11c74634f1afa5bbe0d3f2678b7f819ebf8e/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/managedClusters.json#L1956)does.